src/services/usersService.ts

```diff
@@ -33,7 +33,7 @@ export class UsersService {
      * @param password {string} Password.
      */
     public async signInWithBasic(username: string, password: string): Promise<void> {
-        const credentials = `Basic ${btoa(`${username}:${password}`)}`;
+        const credentials = `Basic ${Buffer.from(`${username}:${password}`, "utf8").toString("base64")}`;
```

Comment:
Are there maybe some constants to use for the "utf8" and "base64" strings? If not LGTM